src/PlatformEngines.jl

```diff
@@ -1226,7 +1226,9 @@ function download_verify_unpack(
         unpack(tarball_path, dest; verbose=verbose)
     finally
         if remove_tarball
-            Base.rm(tarball_path)
+            Base.rm(tarball)
+            # Remove cached tarball hash, if it exists.
+            Base.rm(string(tarball, ".sha256"); force=true)
```

Comment:
`tarball` does not seem to be defined, should it be
```suggestion
            Base.rm(tarball_path)
            # Remove cached tarball hash, if it exists.
            Base.rm(string(tarball_path, ".sha256"); force=true)
```
as the original code?

Reply:
yeah, I don't know what happened.